examples/mouse_position.rs

```diff
@@ -0,0 +1,84 @@
+use bevy::{input::InputSystem, prelude::*, window::PrimaryWindow};
+use leafwing_input_manager::{
+    axislike::DualAxisData, plugin::InputManagerSystem, prelude::*, systems::run_if_enabled,
+};
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_plugin(InputManagerPlugin::<BoxMovement>::default())
+        .add_startup_system(setup)
+        .add_system(
+            update_cursor_state_from_window::<BoxMovement>
+                .run_if(run_if_enabled::<BoxMovement>)
+                .in_base_set(CoreSet::PreUpdate)
+                .in_set(InputManagerSystem::ManualControl)
+                .before(InputManagerSystem::ReleaseOnDisable)
+                .after(InputManagerSystem::Tick)
+                .after(InputManagerSystem::Update)
+                .after(InputSystem),
+        )
+        .add_system(pan_camera)
+        .run();
+}
+
+#[derive(Actionlike, Clone, Debug, Copy, PartialEq, Eq)]
+enum BoxMovement {
```

Comment:
A clearer name of maybe some doc strings would help users understand what this is being used for faster.